docs/contributing.rst

```diff
@@ -199,3 +199,121 @@ The documentation can be built by running::
     $ tox -e docs
 
 The resulting built documentation will be available in the ``.tox/docs/tmp/html`` folder.
+
+Development best practices, policies and procedures
+---------------------------------------------------
+
+The following information is mainly for core developers, but may also be of interest to
+contributors.
+
+Merging pull requests
+~~~~~~~~~~~~~~~~~~~~~
+
+If at all possible, pull requests submitted by an external contributor should be
+reviewed and approved by all core developers before being merged. Pull requests
```

Comment:
Do we want to constraint it to all? It may be difficult to achieve to time constraints and competing demands. Admittedly "all" is small right now though.

Reply:
As contrast, `scikit-learn` requires two approvals for merging (see [Pull Request Checklist]( https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md#pull-request-checklist )). Dask has a more informal mechanism by which core developers say they plan to merge in a particular time frame if they get no comments (typical in a day).

Reply:
I think I'll edit this to say we'd like to aim for pull requests to be reviewed by at least one other core developer. With only two of us at the moment it amounts to the same thing, but I think it's probably worth saying we'd like to aim for some review of PRs. This will slow things down if either of us is unable to review due to other commitments, but I think it's probably ok to move slowly for a while and focus on consolidation and sharing knowledge between core devs.

Reply:
Sure. I completely agree. We also added a lot of features. So we might need to be in maintenance mode for a bit just in case any bugs show up.